src/main/java/duke/data/TaskDataBase.java

```diff
@@ -0,0 +1,88 @@
+package duke.data;
+import java.io.*;
```

Comment:
Perhaps consider only importing the specific package you need, instead of using *